67465 Department Codes (also 96500 Deploy prgrms images)

tyndmar opened the pull request:
Program changed: asiUpdateEnv.w
added logic on every upgrade to set department isRequired flags
added procedure ipSetDepartmentRequired
used list: "PR,GL,QS,WN,WS,FB,FS,RC,RS,CR,PR,CT"
change assignment of menuimage (without extents) to menuimage[1]

Commit:
67465 Department Codes (also 96500 Deploy prgrms images)

Program changed: asiUpdateEnv.w
added logic on every upgrade to set department isRequired flags
added procedure ipSetDepartmentRequired
used list: "PR,GL,QS,WN,WS,FB,FS,RC,RS,CR,PR,CT"
change assignment of menuimage (without extents) to menuimage[1]

jay-advantzware reviewed (commented):
Do not make them active if they have made them inactive already - bDept.isInactive = FALSE.

Commit:
67465 Department Codes

program changed: asiUpdateENV.w - removed requirement for department to be active
